core/src/shred_fetch_stage.rs

```diff
@@ -53,6 +52,7 @@ struct RepairContext {
 
 impl ShredFetchStage {
     // updates packets received on a channel and sends them on another channel
+    #[allow(unused_variables, unused_assignments)]
```

Comment:
Need to keep feature checking machinery for https://github.com/anza-xyz/agave/pull/2149.
This will be removed in https://github.com/anza-xyz/agave/pull/2149.